src/Raven.Client/Documents/Session/Operations/LoadOperation.cs

```diff
@@ -55,6 +63,14 @@ public LoadOperation WithIncludes(string[] includes)
             return this;
         }
 
+        public LoadOperation WithCounters(string[] counters, bool includeAll = false)
```

Comment:
Let's split this to `WithCounters` and `WithAllCounters`

Reply:
done